update contributing.md (#588)
NTMC-Community/MatchZoo: CONTRIBUTING.md

If you've run into behavior in Matchzoo you don't understand, or you're having trouble working out a good way to apply it to your code, or you've found a bug or would like a feature it doesn't have, we want to hear from you!

Our main forum for discussion is the project's [GitHub issue tracker](https://github.com/NTMC-Community/MatchZoo/issues).  This is the right place to start a discussion of any of the above or most any other topic concerning the project.

For less formal discussion we have a chat room on Wechat (mostly Chinese speakers). Matchzoo core developers are almost always present; feel free to find us there and we're happy to chat. Please add *CLJ_Keep* as your wechat friend, she will invite you to join the chat room.

First Time Contributors
-----------------------

Matchzoo appreciates your contribution! If you are interested in helping improve Matchzoo, there are several ways to get started:

* Work on [new models](https://github.com/NTMC-Community/awaresome-neural-models-for-semantic-match).
* Work on [tutorials](https://github.com/NTMC-Community/MatchZoo/tree/master/tutorials).
* Work on [documentation](https://github.com/NTMC-Community/MatchZoo/tree/master/docs).

> * Ask on [the issue tracker](https://github.com/NTMC-Community/MatchZoo/issues) about good beginner issues.

Reviewer: ```suggestion
* Try to answer questions on [the issue tracker](https://github.com/NTMC-Community/MatchZoo/issues).
```
